tools/umi_tools/main.nf

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.preview.dsl = 2
+
+// Local default params
+params.internal_process_name = 'umitools_dedup'
+
+// Process definition
+process umitools_dedup {
+    publishDir "${params.outdir}/${params.internal_process_name}",
+        mode: "copy", overwrite: true
+
+    container 'luslab/nf-modules-umitools:latest'
+
+    input:
+      tuple val(sample_id), path(bai), path(bam)
+       
+    output:
+      tuple val(sample_id), path("*.dedup.bam"), emit: dedupBam
+      path "*.dedup.log", emit: report
+
+    shell:
+
+    // Init
+    internal_prog = "umi_tools dedup"
```

Comment:
Cant we just use this in the command without having a separate parameter for it?

Reply:
yep, can remove this